code/game/objects/items/weapons/handcuffs.dm

```diff
@@ -23,7 +23,7 @@
 	if(!istype(C))
 		return
 
-	if(CLUMSY in user.mutations && prob(50))
+	if(CLUMSY in user.mutations && prob(50) && !istype(src, /obj/item/restraints/handcuffs/toy))

```

Comment:
This is pretty snowflakey as far as code goes. IMO it would be better to handle it like with guns and give handcuffs a variable called ignoreclumsy or something like that and check for it here, rather than a direct istype check.

Reply:
Alright, I'll give that a go tommorow.